### DISCUSSION TITLE
query profile 中active time并不能反映任务总耗时，是否能增加指标去显式的展示每个node执行总耗时?另外profile中的各个指标有没有更详细的解释文档呢？比如PipelineContext

### DISCUSSION THREAD

**Discussion by zhangjjzyl** [apache/doris#32889] [A - General / Q&A]
![image](https://github.com/apache/doris/assets/40597598/bcaacb9f-e581-4c8d-b3f0-6c2686c30d2f)

**Comment by yagagagaga** [comment]
目前profile太复杂，可以使用selectdb开发的webui进行可视化查看

**Reply by zhangjjzyl** [reply]
是什么工具呢？我们装了SelectDB Enterprise这个只有对集群的管理，没有对单个查询的分析

**Reply by yagagagaga** [reply]
https://docs.selectdb.com/docs/enterprise/enterprise-core-guide/selectdb-webui-guide

SelectDB Enterprise 包里面有一个目录叫webui，进入到bin目录下执行启动脚本就可以启动，如果使用了manager部署SelectDB Enterprise ，能够自动识别webui

**Reply by zhangjjzyl** [reply]
我进入webUI页面了，但是找不到文档上的查看执行计划的位置，在文档也没找到需要配置的地方
安装版本`selectdb-doris-2.0.4`
我的页面
![image](https://github.com/apache/doris/assets/40597598/90f11619-e98e-418b-b74e-bb8d01432c9d)
官方文档
![image](https://github.com/apache/doris/assets/40597598/0f5ac5f5-9444-4468-b4c9-388a36ac8e0f)
